### ISSUE TITLE
Has the minimum SDK for flutter changed now ?

### ISSUE THREAD

**Issue by sallypeters** [miguelpruivo/flutter_file_picker#381]
Hi - previously - I've been running without fail a branched version of file_picker located here :

 url:  https://github.com/miguelpruivo/flutter_file_picker.git 
     ref:  fix/open_document 
     path: file_picker/

This branch fixed an issue that I was also having with regards to opening a document in 1 Drive. 

It now seems that something has changed because I get an immediate crash in Android : 

```
Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
com.mr.flutter.plugin.filepicker.FilePickerDelegate$2.run

```

I have also tried 1.13.3 and have the same issue. I cannot upgrade to Flutter 1.17 because of one or more issues that I have with plugins that I am using.

I also now notice that in Crashalytics - I also seem to be seeing this message at the time File_Picker kicks in : 

'Only fullscreen activities can request orientation flutter'

**Comment by miguelpruivo** [comment]
Not actually @sallypeters. Must be something related with your app or some other dependency I’d say.

Let me know if I can assist you with something else.

**Comment by sallypeters** [comment]
@miguelpruivo - I've created a brand new Flutter App pointing at the SDK I'm using and still get the same issue.

I appear to be having this exact same issue mentioned here : 

App crash on file pick in android #313
Flutter app crashes whenever a file is chosen. #339

The last thread mentions that users need to disable an option in 'Developer Options'  (Which assumes they have made themselves a developer in the first place - I cant see normal users doing this to get an app to work and how would they even know this is the solution?) - I tried this and still have the same issue.

**Comment by miguelpruivo** [comment]
@sallypeters this was a known issue prior to Flutter 1.17 which evaluates a null object in a place where it can’t be.

I’m afraid there is no way around it because that’s actually a bug with the SDK which was in the meantime fixed (hence, the 1.17 version is required). 

Flutter currently is on stable is on 1.20. If you can’t upgrade it for some reason, I suggest you to use a direct git reference for a commit previous of that feature as dependency and use it like so. 

I’m sorry for any inconvenience. I’m closing this as it was already fixed, but we can still further discuss it here.

Thank you!
